common/app/model/dotcomrendering/DotcomRenderingUtils.scala

```diff
@@ -30,9 +30,42 @@ import views.support.AffiliateLinksCleaner
 
 import java.net.URLEncoder
 
+sealed trait DotcomRenderingMatchType
+
+object DotcomRenderingMatchType {
+  implicit val matchTypesWrites = new Writes[DotcomRenderingMatchType] {
+    def writes(matchType: DotcomRenderingMatchType) =
+      JsString(matchType.toString)
+  }
```

Comment:
Scala was unable to implicitly determine the JSON respresentation of this object with 

`implicit val writes: Writes[DotcomRenderingMatchType] = Json.writes[DotcomRenderingMatchType]` 

So I used this `toString` method which I saw here: https://github.com/guardian/frontend/blob/main/common/app/model/meta.scala#L243